nipype/interfaces/ants/segmentation.py

```diff
@@ -266,10 +266,15 @@ class N4BiasFieldCorrectionInputSpec(ANTSCommandInputSpec):
     input_image = File(
         argstr='--input-image %s',
         mandatory=True,
-        desc=('image to apply transformation to (generally a '
-              'coregistered functional)'))
-    mask_image = File(argstr='--mask-image %s')
-    weight_image = File(argstr='--weight-image %s')
+        desc=('input for bias correction. Negative values or values close to '
+            'zero should be processed prior to correction'))
```

Comment:
`processed` - how? if there is some additional info we can add that would be great.

Reply:
I don't even know (yet).  This was copy pasted from the --help output of the tool and IMHO is better than nothing ;-)